### PR TITLE
Use defcustom and Change Some Wording

### PR DESCRIPTION
**Pull request by LaurenceWarne**
Hi! I think this package is really neat. I've put together this PR which changes `string-inflection-skip-backward-when-done` to be defined via `defcustom`.

I've also changed some of the wording in the README so it's closer to English language norms - I hope this is useful and not too intrusive :slightly_smiling_face:

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -84,7 +84,7 @@ The =string-inflection-all-cycle=, conversion often. However,
 conversion occurs even when there is no need to lower-camelcase.
 Therefore so difficult to use, is not recommended
```
Comment (LaurenceWarne) [inline README.org:85]: Is this comment still accurate after https://github.com/akicho8/string-inflection/commit/4df4b62666991d6f57fe8968b20a8f938bade888?

Comment (akicho8) [reply README.org:85]: Accurate.
I still do not use `string-infection-all-cycle`.
It is better to use the appropriate function to match the emacs major-mode.

**Comment by akicho8** [comment]
Thanks to this PR.
I know only Japanese language, so I was anxious that the translated English reading was read and understood.
I am glad to fix it to the English that everyone can understand.

File: README.org (hunk 1 above)
Comment (LaurenceWarne) [reply README.org]: >  It is better to use the appropriate function to match the emacs major-mode.

**Comment by LaurenceWarne** [comment]
> Thanks to this PR.
> I know only Japanese language, so I was anxious that the translated English reading was read and understood.
> I am glad to fix it to the English that everyone can understand.

No problem, mostly the meaning comes across fine, only the wording is a little strange :slightly_smiling_face:

**Pull request merged**